codecov.yml

```diff
@@ -4,3 +4,10 @@ comment:
   require_changes: false
   require_base: no
   require_head: yes
+
+coverage:
+  status:
+    project:
+      default:
+        if_ci_failed: success
```

Comment:
I don't think we want this. Doesn't this mean if a unit test fails, we still pass CI?

I *think* this is for cases where you run unittests separately from code coverage.

Reply:
You're right. Since you have this in your PR #6 , I will remove the updates of yml here.

Reply:
Will rebase onto latest main after you merge it.